k9mail/src/main/res/layout/activity_about.xml

```diff
@@ -17,7 +17,7 @@
 
         <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
+            android:layout_height="wrap_content"
```

Comment:
is this for scrollers?

Reply:
Yes, a layout inside a ScrollView should have the free height just to wrap its content.